### PR TITLE
Add Russian and Ukrainian pluralizer languages

### PR DESCRIPTION
**Pull request by kostya9**
Adding Ukrainian and Russian pluralizer languages. The rules were taken from this table https://unicode-org.github.io/cldr-staging/charts/37/supplemental/language_plural_rules.html

Added tests.

### CODE REVIEW

File: src/Jeffijoe.MessageFormat.Tests/Formatting/Formatters/PluralFormatterTests.cs [hunk 1]
```diff
@@ -74,6 +74,56 @@ public void ReplaceNumberLiterals(string input, string expected)
             var actual = subject.ReplaceNumberLiterals(new StringBuilder(input), 1337);
             Assert.Equal(expected, actual);
         }
+        
+        [Theory]
+        [InlineData(0, "днів")]
+        [InlineData(101, "день")]
+        [InlineData(102, "дні")]
```
Comment (kostya9) [inline src/Jeffijoe.MessageFormat.Tests/Formatting/Formatters/PluralFormatterTests.cs:81]: Here I tested in the same language the pluralizer is for, it's easier to see whether the rules are correct for a native speaker

File: src/Jeffijoe.MessageFormat/Formatting/Formatters/PluralFormatter.cs [hunk 2]
```diff
@@ -293,6 +293,70 @@ private void AddStandardPluralizers()
...
+                            (remainderHundred >= 11 && remainderHundred <= 14))
+                        {
+                            return "many";
+                        }
+                    }
+
+                    return "other";
+                });
+            
+            this.Pluralizers.Add(
```
Comment (kostya9) [inline src/Jeffijoe.MessageFormat/Formatting/Formatters/PluralFormatter.cs:329]: uk and ru rules are incidentally the same, but decided not to reuse the function, as they may be refactored to multiple files later

**Comment by jeffijoe** [comment]
Very nice!

In #22 we were discussing other locales. I wonder what would be the best course of action to allow users to easily add the languages that they need (and contribute them back) while still keeping the runtime lean for those who don't need every language contributed. 🤔 

Having a NuGet package per language is obviously not scalable, but perhaps refactor so instead of adding to a `Pluralizers` array we introduce a `LocaleCollection` structure that can be constructed and passed into instances of `MessageFormatter`.

What do you think?

**Comment by kostya9** [comment]
I was thinking of the same thing, I thought of using a text file with rules. https://github.com/google/libphonenumber uses it for phone formats  (see an example PR that adds a mobile phone here https://github.com/google/libphonenumber/pull/2567/files - looks easy contribute to). And, additionally, leaving the delegate for some very custom logic should be fine too

**Comment by jeffijoe** [comment]
I left a comment in #22 about this; feel free to join that conversation, too. I'm gonna hold off on merging this for a bit to see if something comes up that we can implement right away. Is that ok with you?

If nothing happens on that front then I'll merge this as-is, just want to give it a week or 2. :)

**Comment by kostya9** [comment]
Sure, will add these languages manually for now

**Comment by kostya9** [comment]
Closing in favour of something more systematic, meaning for all locales. That is being discussed in https://github.com/jeffijoe/messageformat.net/issues/22

**Pull request closed without merging**